moengage/src/main/java/com/rudderstack/android/integrations/moengage/MoengageIntegrationFactory.java

```diff
@@ -74,8 +76,10 @@ private void processRudderEvent(RudderMessage element) {
                         return;
                     } else if (event.equals("Application Installed")) {
                         helper.setAppStatus(context, AppStatus.INSTALL);
+                        return;
```

Comment:
Can you please explain the reason for returning from here ?

Because, I could see that previously we are setting the app status as well as tracking an event in case of both Application Installed and Updated.  Are we seeing any duplicate events on Moengage dashboard in case of Application Install/Update ?

Reply:
Two events  are being fired `INSTALL / Device ReInstall` and `Application Installed` events (from MoEngage device mode) whenever corresponding lifecycle events are triggered.
Since we are are already making an specific API call for lifecycle events (by this I mean `setting the app status`), shouldn't we drop the other (custom lifecycle events call)?
This is the reason, I decided to drop the other (custom lifecycle) call.
Please LMK if we need to make both calls?

By duplicate, I mean that two similar type of events are appearing in the MoEngage dashboard.

Also, I've tested this behaviour on iOS and attached the screenshot for reference, where we could see `INSTALL` and `Application Installed` events appeared at the same time (which I thought causing duplicacy):
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/64667840/209767460-fdab1ec8-83b2-4bc3-835a-10900a5d7e20.png">

Reply:
If we stop sending `Application Installed` / `Application Updated` it would be a breaking change, because customers might be depending on these events already right. 

I feel that we should make both the calls, because for `Application Backgrounded` we are not mapping it to any of the MoEngage's API's, and at the end customer would be able to see only one event out of all the three life cycle events tracked automatically by us, and it might cause some confusion. 

Let's also wait for @pallabmaiti  and @itsdebs thoughts as well.

Reply:
Summary: We decided not to sent custom lifecycle events, as in our other destinations e.g., Firebase we aren't doing so.